aten/src/ATen/function_wrapper.py

```diff
@@ -1203,16 +1205,16 @@ def gen_namespace_function(option, multidispatch_tensors):
         # we just implement it in the base Type.  This is exposed
         # in Declarations.yaml via a field named 'abstract'.
         abstract = False
+        op_registrations.append(OpRegistration(
```

Comment:
Why this change?

Reply:
I need to register schemas EVEN if there is a dictionary type method dispatch. Because those individual dictionary entries aren't going to have schemas associated with them.

Actually, I had to fix this code up some more in later PR, so the latest version is now:

```
        abstract = False
        if isinstance(type_method_dispatch, dict):
            abstract = True
            # Having manual_kernel_registration for an abstract method doesn't make sense.
            assert not option['manual_kernel_registration']
            op_registrations.append(OpRegistration(
                operator_name=OPERATOR_NAME.substitute(option),
                registration_code=SCHEMA_REGISTRATION.substitute(option)))
        else:
            top_env['type_method_declarations'].append(NATIVE_DISPATCH_DECLARATION.substitute(option))
            top_env['type_method_definitions'].append(NATIVE_DISPATCH_DEFINITION_DEFAULT.substitute(option))
            if not option['manual_kernel_registration']:
                op_registrations.append(OpRegistration(
                    operator_name=OPERATOR_NAME.substitute(option),
                    registration_code=SCHEMA_REGISTRATION.substitute(option)))
                if option['use_c10_dispatcher'] == 'full':
                    op_registrations.append(OpRegistration(
                        operator_name=OPERATOR_NAME.substitute(option),
                        registration_code=DEFAULT_FUNCTION_REGISTRATION.substitute(option)))
                else:
                    assert option['use_c10_dispatcher'] == 'unboxed_only'
                    op_registrations.append(OpRegistration(
                        operator_name=OPERATOR_NAME.substitute(option),
                        registration_code=DEFAULT_UNBOXEDONLY_FUNCTION_REGISTRATION.substitute(option)))
```